pages/common/ar.md

```diff
@@ -3,22 +3,22 @@
 > Create, modify, and extract from archives (`.a`, `.so`, `.o`).
 > More information: <https://manned.org/ar>.
 
-- Extract all members from an archive:
+- E[x]tract all members from an archive:
 
-`ar -x {{path/to/file.a}}`
+`ar x {{path/to/file.a}}`
 
-- List the members of an archive:
+- Lis[t] contents in a specific archive:
 
-`ar -t {{path/to/file.a}}`
+`ar t {{path/to/file.a}}`
 
-- Replace or add files to an archive:
+- [r]eplace or add specific files to an archive:
 
-`ar -r {{path/to/file.a}} {{path/to/file1.o}} {{path/to/file2.o}}`
+`ar r {{path/to/file.a}} {{path/to/file1.ext path/to/file2.ext ...}}`
 
-- Insert an object file index (equivalent to using `ranlib`):
+- In[s]ert an object file index (equivalent to using `ranlib`):
 
-`ar -s {{path/to/file.a}}`
+`ar s {{path/to/file.a}}`
 
-- Create an archive with files and an accompanying object file index:
+- Create an archive with specific files and an accompanying object file index:
 
-`ar -rs {{path/to/file.a}} {{path/to/file1.o}} {{path/to/file2.o}}`
+`ar rs {{path/to/file.a}} {{path/to/file1.ext path/to/file2.ext ...}}`
```

Comment:
As far as I know, creating an **object file** index only makes sense, when you have object files.

```suggestion
`ar rs {{path/to/file.a}} {{path/to/file1.o path/to/file2.o ...}}`
```